GOBIN must be an absolute path

# Problem

Actually Makefile is overwriting GOBIN environment variable to be `build/bin`. This doesn't work with any `go get ..` commands as GOBIN must refer to an absolute path. This problem causes `go get` commands to crash when they run through the 

```
$ make xgo
docker pull statusteam/xgo:1.9.2
1.9.2: Pulling from statusteam/xgo
Digest: sha256:d7f275998ec52a2a0c19a223115a6aaf67ed6f5fa0bf327c41af54e8def22292
Status: Image is up to date for statusteam/xgo:1.9.2
build/bin
go get github.com/karalabe/xgo
cannot install, GOBIN must be an absolute path
make: *** [xgo] Error 1
```

# Implementation

Path should be completed with current makefile's own path, so we have a valid absolute path.

# Acceptance Criteria

- [x] Dependant `make xgo` make endpoints should keep working. 
- [x] Status-React related commands should also be working `scripts/bundle-status-go.sh android`

--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,8 @@ endif
 
 CGO_CFLAGS=-I/$(JAVA_HOME)/include -I/$(JAVA_HOME)/include/darwin
 BUILD_TAGS =
-GOBIN = build/bin
+GOBIN=$(dir $(realpath $(firstword $(MAKEFILE_LIST))))build/bin
+
 GO ?= latest
 XGOVERSION ?= 1.9.2
 XGOIMAGE = statusteam/xgo:$(XGOVERSION)

